docs/rst/tutorials/micro/user_api/user_api.rst

```diff
@@ -0,0 +1,19 @@
+.. _tutorials_micro_user_api:
+
+micro-ROS User API
+=======================
+
+In this section, you’ll learn the basics of the micro-ROS C API: ``rclc``. This API implementation is based on the ROS 2 client support library (rcl), enriched with a set of convenience functions by the package rclc.
+That is, rclc does not add a new layer of types on top of rcl (like RCLCPP and RCLPY do) but only provides functions that ease the programming with the rcl types. New types are introduced only for concepts that are missing in rcl, such as the concept of an executor.
+
+This API implements the embedded version of the complete ROS 2 communications paradigm:
+
+.. toctree::
+    :maxdepth: 1
+
+    user_api_nodes.rst
+    user_api_pub_sub.rst
+    user_api_services.rst
+    user_api_executor_timers.rst
+
+.. TODO(acuadros95): Move ROS2 documentation references to future Vulcanexus tutorials.
```

Comment:
Can you solve this now that we have the ROS 2 docu integrated?

Reply:
Yes, solved here: 456cd803dec6c4a5d826ad6c35d6c0c4d540eb91

This PR needs to be merged first though: https://github.com/eProsima/ros2_documentation/pull/6